flintrock/core.py

```diff
@@ -140,8 +140,8 @@ def load_manifest(self, *, user: str, identity_file: str):
             manifest_raw = ssh_check_output(
                 client=master_ssh_client,
                 command="""
-                    cat /home/{u}/.flintrock-manifest.json
-                """.format(u=shlex.quote(user)))
```

Comment:
Why are we removing the `shlex.quote()`? It's there to protect against funny business with the user name.